views/showcase.html

```diff
@@ -339,6 +339,14 @@ <h2 id="readme" class="section-title">Read me</h2>
     var query = document.getElementById('search-input').value;
     window.location.href = '/search?q=' + query;
   }
+
+  function sendEvent(fileFormat, eventCategory, resourceTitle) {
```

Comment:
I think we should use `resourceName` not `resourceTitle` because `title` is an optional property, while `name` is an identifier.